crescent/internal/app_command.py

```diff
@@ -35,7 +35,7 @@ def __attrs_post_init__(self) -> None:
 
     @classmethod
     def from_meta_struct(
-        cls: Type[Unique], command: MetaStruct[CommandCallbackT, AppCommandMeta]
+        cls: type[Unique], command: MetaStruct[CommandCallbackT, AppCommandMeta]
```

Comment:
The reason this is failing is because (mypy thinks, at least) `type` is defined as a class var on line 23 (and it might be?).

Reply:
tested. This is why.

Reply:
This is why we never shadow builtins :pensive:

Maybe leave this as typing.Type for now and fix the property name seperately?

Reply:
could try making it a string? Not too familiar with how that'll work though.

Reply:
Making it a string won't fix it. Mypy will still think it references the `type` classvar on the class.

Reply:
I think for now this just needs to be left as typing.Type, and we can refactor the class to not shadow builtins later.